app/core/api/users.js

```diff
@@ -232,5 +232,13 @@ module.exports = {
   },
   getLowUsageData (userId, options) {
     return fetchJson(`/db/users/${userId}/low-usage-data`, options)
-  }
+  },
+
+  fetchClassNamesForUser (ids) {
+    return fetchJson('/db/classrooms/-/names/by-user', {
```

Comment:
you know it would be apt to rename it to `db/users/classroom-names` or something like that too 😅